platform/windows/godot_res.rc

```diff
@@ -21,7 +17,7 @@ BEGIN
             VALUE "FileVersion",            VERSION_NUMBER
             VALUE "ProductName",            VERSION_NAME
             VALUE "Licence",                "MIT"
-            VALUE "LegalCopyright",         "Copyright (c) 2007-" _MKSTR(VERSION_YEAR) " Juan Linietsky, Ariel Manzur and contributors"
+            VALUE "LegalCopyright",         "Copyright (c) 2007-present Juan Linietsky, Ariel Manzur and Godot Engine contributors"
```

Comment:
I wondered how the copyright line appears on Windows, and whether the "Copyright (c)" part would be redundant with Windows' own labels, or whether it needs to be kept short. Could be good if someone can check on a Windows build.

Reply:
It is redundant, but it's common way to do it.

<img width="622" alt="yyy" src="https://github.com/godotengine/godot/assets/7645683/ef00e570-7da3-4a8f-a4d6-0a7c5e96f52f">

Reply:
I shortened it a bit by removing the "Copyright" word, but keeping "(c)".